sample_metadata/parser/generic_parser.py

```diff
@@ -73,7 +73,7 @@
 
 # construct rmatch string to capture all fastq patterns
 rmatch_str = (
-    r'[_\.-][Rr]?[12]('
+    r'[_\.-][Rr]?[12](_\d+)?('
```

Comment:
It looks like this change got included in this PR, not a big deal, but may cause a conflict when the original PR is merged